handlers/main.yml

```diff
@@ -9,6 +23,7 @@
     - assets:clobber
     - assets:precompile
     - db:migrate
+    - db:seed
```

Comment:
I actually think `db:seed` is dangerous, and we should not do it in production.

Reply:
it was running in production on the current tul_cob playbook.

Reply:
removed.